catalog.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Каталог HTML Academy: Нёрдс</title>
+  </head>
+  <body>
+    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p> 
+    <header>
+      <nav>
+        <a>
+          <img src="img/index-logo.svg" width="368" height="153" alt="Логотип студии «Нёрдс»">
+        </a>
+        <ul>
+          <li><a href="blank.html">Студия</a></li>
+          <li><a href="blank.html">Клиенты</a></li>
+          <li><a href="catalog.html">Магазин</a></li>
```

Comment:
Это  активный пункт меню. Страница ссылаться на себя не должна, поэтому `href` удаляем:
```suggestion
          <li><a>Магазин</a></li>
```